aquatic_udp/src/workers/request/mod.rs

```diff
@@ -15,7 +15,7 @@ use aquatic_udp_protocol::*;
 use crate::common::*;
```

Comment:
Were these functions made public to support fuzzing?

Reply:
yes, some functions and types as well. I wanted to fuzz incoming UDP packets without IO, but also without modifying much of their code. Exporting types was the least intrusive way to do that.

That said, my impression of this project is that it is a binary, and the library crates are not realistically intended for external use. This is a common strategy to speed up compile times for projects with lots of code. So we can see what upstream thinks, but they could be totally fine with it. If not, I could put the harness code (in fuzz_targets) in the library crate exposed only under a `#[cfg(fuzzing)]`